sample/src/main/assets/json.form/basic_form.json

```diff
@@ -65,6 +65,39 @@
         "type": "choose_image",
         "uploadButtonText": "Take a photo of the child"
       },
+      {
+        "key": "finger_print",
```

Comment:
This two fields have the same key I don't think the updates will work well here.

Reply:
Actually it's sample how we need to add register and verify finger print option. there had a key like "finger_print_option":"register", or "finger_print_option":"verify",